numpy/ma/tests/test_core.py

```diff
@@ -1868,16 +1868,25 @@ def test_extremum_fill_value(self):
         assert_equal(test['B']['BB'], default_fill_value(a['B']['BB']))
 
         test = minimum_fill_value(a)
-        assert_equal(test[0], minimum_fill_value(a['A']))
-        assert_equal(test[1][0], minimum_fill_value(a['B']['BA']))
-        assert_equal(test[1][1], minimum_fill_value(a['B']['BB']))
-        assert_equal(test[1], minimum_fill_value(a['B']))
+        assert_equal(test['A'], minimum_fill_value(a['A']))
+        assert_equal(test['B']['BA'], minimum_fill_value(a['B']['BA']))
+        assert_equal(test['B']['BB'], minimum_fill_value(a['B']['BB']))
+        assert_equal(test['B'], minimum_fill_value(a['B']))
```

Comment:
I'd be incredibly surprised if any user in the wild calls `minimum_fill_value` in the first place